src/components/playback/playbackmanager.js

```diff
@@ -2580,16 +2580,15 @@ export class PlaybackManager {
                 });
             }
 
-            const apiClient = ServerConnections.getApiClient(item.ServerId);
-            let mediaSourceId;
-
-            const isLiveTv = [BaseItemKind.TvChannel, BaseItemKind.LiveTvChannel].includes(item.Type);
-
-            if (!isLiveTv) {
-                mediaSourceId = playOptions.mediaSourceId || item.Id;
+            let mediaSourceId = playOptions.mediaSourceId;
+            const needsTrackWorkaround = (playOptions.audioStreamIndex || playOptions.subtitleStreamIndex);
```

Comment:
We should do it after `autoSetNextTracks` ([here](https://github.com/jellyfin/jellyfin-web/blob/6fd90b0c368bc8dc460fa7b404d47d9928b9be12/src/components/playback/playbackmanager.js#L2626)), like how it was done [before](https://github.com/viown/jellyfin-web/blob/95e23875f4e2761a8ed65d73dd5b94977e0d0d79/src/components/playback/playbackmanager.js#L2614).